christmas/styles.css

```diff
@@ -0,0 +1,344 @@
+@import url(https://fonts.googleapis.com/css?family=Allan:700,400);
+
+@keyframes new {
+    0% {
+        left: 0%;
+        margin-left: -550px;
+    }
+    100% {
+        left: 50%;
+        margin-left: -275px;
+    }
+}
+@keyframes open {
+    0% {
+        transform: translate3d(0, 0, 0) rotateY(0);
+    }
+    33.333% {
+        transform: translate3d(-100%, 0, 0) rotateY(-180deg);
+    }
+    66.666% {
+        transform: translate3d(-100%, 0, 0) rotateY(-180deg);
+    }
+    100% {
+        transform: translate3d(-100%, 733.33333px, 0) rotateY(-180deg);
+    }
+}
+@keyframes flap-open {
+    0% {
+        transform: rotateX(0deg);
+        z-index: 5;
+    }
+    50% {
+        transform: rotateX(0deg);
+        z-index: 5;
+    }
+    100% {
+        transform: rotateX(-180deg);
+        z-index: -1;
+    }
+}
+@keyframes letter-out {
+    0% {
+        transform: translate3d(0, 0, 0);
+        box-shadow: inset 0 0px 30px -5px #b08c5b, 0 0 10px -5px;
+    }
+    100% {
+        transform: translate3d(0, -733.33333px, 0);
```

Comment:
In mobile devices the envelope is still partially visible. Is this intentional? 
![image](https://user-images.githubusercontent.com/20641536/102878495-44ee5d80-4440-11eb-99d5-17cfa9bd90dd.png)

Reply:
This is intentional 👍